specs/order_spec.rb

```diff
@@ -82,29 +82,49 @@
 xdescribe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      Grocery::Order.all.length.must_equal 100
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+      id = 1
+      food = { "Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9 }
+      order = Grocery::Order.new(id, food)
+      order.id.must_equal id
+      expected_count = 3
+      order.products.count.must_equal expected_count
+      order.products.must_include "Grape Seed Oil"
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      id = 100
+      food = {"Allspice" => 64.74, "Bran" => 14.72, "UnbleachedFlour" => 80.59 }
+      order = Grocery::Order.new(id, food)
+      order.id.must_equal id
+      expected_count = 3
+      order.products.count.must_equal expected_count
+      order.products.must_include "UnbleachedFlour"
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+      id = 1
+      all = Grocery::Order.find(1)
+      all.find id must_equal id
```

Comment:
These tests are on the right track. One observation is that `Order.find` doesn't return all the orders, it returns a single order. So a better implementation might be:

```ruby
# Arrange
id = 1

# Act
order = Grocery::Order.find(id)

# Assert
# Make sure we actually got back an Order
order.must_be_kind_of Order
# If we got an order back with a different ID, that's a bug
order.id.must_equal id
```